src/pyedb/configuration/cfg_components.py

```diff
@@ -67,6 +68,23 @@ def __init__(self, **kwargs):
 
         self.rlc_model = [CfgRlcModel(**rlc_m) for rlc_m in rlc_models]
 
+        self.x_location, self.y_location = kwargs.get("location", [None, None])
+        self.angle = kwargs.get("angle", None)
+        self.placement_layer = kwargs.get("placement_layer", None)
+
+    def export(self):
```

Comment:
What is the scope of this `export` method ? Can you rename it or add docstring such that we know why it doesn't contain all attributes data ?